### ISSUE TITLE
DOUBT ABOUT TRAILING ON LOWBB STRATEGY BUY

### ISSUE THREAD

**Issue by victorlmt1** [taniman/profit-trailer#1365]
Hello guys, let me ask you a question.

When I use LOWBB as buy strategy and set 

ALL_trailing_buy = 1

Does it means that PT will trail 1% of the distances between BollingerBands or 1% of the coin value?

I think this making my bot miss the buying time, I consider this 1% of trailing based on the formula for calculating the BB value.

As example:

ALL_buy_strategy = LOWBB
ALL_buy_value = 20
ALL_trailing_buy = 1

The PT start to trail certain coin that hits 20% above lowbb.
The PT keeps trailing it and it hits 15% above lowbb as the lowest position.
The PT buys it when it hits 16,1% above lowbb because ALL_trailing_buy is set to 1%.

Am I correct? Thanks.

**Comment by Utone** [comment]
not sure think its % based indeed you can miss out on low dips very easy, let me show you my approach, when dip buying i disable trailing completely but first you look for them:

BB tips:
In bleeding, red, bearish markets:  try -10BB to even -22BB 
To fine-tune dip buying levels and my preferred method while actively trading and tweaking everything in a slight BTC rise (also can be used in normal market) but it requires adjustments so you need to stay present ok or use a trailing buy and less tight /optimal settings, which still will profit but not at all times. 

-Set a trailing buy and enable SOM  
-Now check if true pairs pop up, and dial down bit by bit even possibly as low as -20 until there are only 1 or 2 at a time this is how you check where the dips are at.

[When tuned tight and optimally for the lowest buys at the current market condition: ]

-DISABLE Trailing Buy this way, now you buy the coins when they enter that low area while if you use  trailing buy you wait for someone to snatch it away in front of your sorry ass face. ;)
- Now disable SOM, so buys will and trading will start

(This varies with BTC price, Tip: open up a tradingview window with USDBTC (picture included)
When it starts rising you can set it lower when it drops adjust a bit higher.

Sometimes you can stay at the same buy setting for 2 hours but this might change instantly the visual gives you a good indication, for you to act on, in case the market shifts..
When ALTS start to rise the included default settings with enabled trailing buy are pretty safe and less maintenance is required. 

(On the other alts rising like crazy side: higher BB levels from 35 to 45 can be optimal. I use that with a trailing buy around  0.5-1)

**Comment by victorlmt1** [comment]
> -DISABLE Trailing Buy this way, now you buy the coins when they enter that low area while if you use trailing buy you wait for someone to snatch it away in front of your sorry ass face. ;)

LMFAO, hehehe, that's exactly what happened with me past few days.

I have decided to trade only USDTBTC, aiming to get 1 - 2% profit a day, that's my goal.

I'm using only LOWBB strategy because I used BB when I was trading without a BOT and it works fine. However, I've seen a lot of opportunities to buy but the bot wasn't buying.

I think i'm going to put someone to stay in front of the monitor looking at tradingview and keep adjusting BB value without a trailing buy.

I really appreciate your answer.

Good luck at your trades =)

CYA
